app/frontend/components/common/banner.js

```diff
@@ -0,0 +1,32 @@
+const {h} = require('preact')
+
+const Banner = ({closeBanner}) =>
```

Comment:
I would not make a generic banner element if it is used only once. If it is part of the login page, I would rename it to `StakingBanner` and put it in `components/login`.

Reply:
Fixed